### DISCUSSION TITLE
ESP32 bluetooth && MISCALEV2

### DISCUSSION THREAD

**Discussion by jorgekramer** [arendst/Tasmota#15524] [Q&A] [answered]
Surprise: MISCALEV2 also works on Tasmota v11.1.0 Ostara. Unfortunately weight data is available only as integer value, wishing 2 digit decimal to be more precise.

**Comment by barbudor** [comment, marked as answer]
Have you tried [WeighRes](https://tasmota.github.io/docs/Commands/#weightres) ?

**Reply by jorgekramer** [reply]
Thanks for the fast reply. WeightRes did It. Problem solved
